apps/deploy-web/src/components/sdl/SimpleServiceFormControl.tsx

```diff
@@ -219,59 +229,116 @@ export const SimpleServiceFormControl: React.FunctionComponent<Props> = ({
                         control={control}
                         name={`services.${serviceIndex}.image`}
                         rules={{
-                          required: "Docker image name is required.",
-                          validate: value => {
-                            const hasValidChars = /^[a-z0-9\-_/:.]+$/.test(value);
```

Comment:
We could use the `register` function on the input component if we want to keep the validation instead of the `Controller` component.